docs/blog/2025-11-11-webforj-v25.10/25.10.md

```diff
@@ -0,0 +1,107 @@
+---
+title: What's new in version 25.10?
+description: Get to know the features, fixes, and functionality new in webforJ version 25.10.
+slug: whats-new-v25.10
+date: 2025-11-11
+authors: webforJ
+tags: [webforJ, v25.10, release]
+image: "https://cdn.webforj.com/webforj-documentation/blogs/webforj-v25.10/cover.png"
+hide_table_of_contents: false
+---
+
+![cover image](https://cdn.webforj.com/webforj-documentation/blogs/webforj-v25.10/cover.png)
+
+webforJ version `25.10` introduces some heavy hitting new capabilities for building global, secure, and integrated applications. This release brings internationalization support, enterprise-grade security which includes Spring Security integration, Webswing compatibility for legacy Java Swing apps, and adds to our `Login` component.
+
+<!-- truncate -->
+
+As always, see the [GitHub release overview](https://github.com/webforj/webforj/releases/tag/25.10) for a more comprehensive list of changes. Highlighted below are some of the most exciting changes:
+
+## Route Security with Spring Security integration
```

Comment:
Let's use this video from the CDN: webforj-documentation/blogs/webforj-v25.10/spring-sequirty.mov